<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNaczepa</class>
 <widget class="QDialog" name="AddNaczepa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Data wygaśnięcia przeglądu</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDateEdit" name="dateEditPrzeglad"/>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEditOs3_2">
     <property name="text">
      <string>Liczba kilimetrów na osi 3</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditOs4">
     <property name="text">
      <string>Liczba kilimetrów na osi 4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDateEdit" name="dateEditOs5"/>
   </item>
   <item row="4" column="1">
    <widget class="QDateEdit" name="dateEditOs4"/>
   </item>
   <item row="4" column="0">
    <widget class="QDateEdit" name="dateEditOs3_2"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>DataWygaśnięcia Tachografu</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data wymiany osi4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Data wymiany osi 3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Data wymiany osi 5</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnuluj">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEditOs5">
     <property name="text">
      <string>Liczba kilimetrów na osi 5</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEditNumerRej">
     <property name="text">
      <string>Numer Rejestracyjny</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDateEdit" name="dateEditTachograf"/>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="spinBoxIdKierowca"/>
   </item>
   <item row="9" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>ID Kierowcy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
